Covid.xcodeproj/project.pbxproj

```diff
@@ -796,7 +874,8 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_ENTITLEMENTS = Covid/Covid.entitlements;
-				CODE_SIGN_STYLE = Automatic;
+				CODE_SIGN_IDENTITY = "Apple Development";
+				CODE_SIGN_STYLE = Manual;
```

Comment:
manual kvoli bitrisu, ci len aby nevznikal nahodny bordel?

Reply:
To druhe :)